Rakefile

```diff
@@ -2,7 +2,6 @@ require 'rubygems' if RUBY_VERSION < '1.9.0'
 require 'puppetlabs_spec_helper/rake_tasks'
 require 'puppet-lint/tasks/puppet-lint'
 require 'puppet-syntax/tasks/puppet-syntax'
-require 'metadata-json-lint/rake_task'
```

Comment:
That's odd - any idea why they wouldn't run on your machine?  I'd like to keep it as it helps find when I mess up a comma in the metadata.